physical_ai_server:
  ros__parameters:
    omx:
      observation_list:
        - cam_wrist
        - state

      camera_topic_list:
        - cam_wrist:/camera/cam_wrist/color/image_rect_raw/compressed

      joint_topic_list:
        - follower:/joint_states
        - leader:/leader/arm_with_timestamp

      joint_list:
        - leader

      joint_order:
        leader:
          - joint1
          - joint2
          - joint3
          - joint4
          - gripper
